### ISSUE TITLE
Add support for "latest" as a version number in package URLs

### ISSUE THREAD

**Issue by pushfoo** [gren-lang/package.gren-lang.org#38]
**TL;DR:** Add support for "latest" as a version in package URLs

### What should be added

**TL;DR:** Support `https://packages.gren-lang.org/package/blaix/gren-ansi/version/$VERSION_STRING/overview`

At the moment, the latest package and a version number use different URL formats:

| Package Version | URL Format                                                                                                          |
|------------------------|----------------------------------------------------------------------------------------------------------|
| Latest                  | https://packages.gren-lang.org/package/blaix/gren-ansi/latest/overview              |
| 1.0.0                    | https://packages.gren-lang.org/package/blaix/gren-ansi/version/1.0.0/overview  |

### Why?

1. [At least 2 devs on Discord](https://discord.com/channels/1250584603085766677/1250590753806356571/1254401960904364104) have found this to be a stumbling block
2. @robinheghan asked [for this to be filed on Discord](https://discord.com/channels/1250584603085766677/1250590753806356571/1254472769089835079)
3. It may simplify the backend code
